src/wait/src/lib.rs

```diff
@@ -11,7 +11,7 @@ use resource::rusage;
 
 #[no_mangle]
 pub unsafe extern "C" fn wait(stat_loc: *mut c_int) -> pid_t {
-    unimplemented!();
+    waitpid(0 - 1, stat_loc, 0)
```

Comment:
It is better to use `!0` rather than `0 - 1`, as that implies an overflow for usize.

Reply:
Does that have the same effect? The posix spec says it should be (pid_t)-1

Reply:
Yes